ubuntu-advantage

```diff
@@ -71,9 +16,16 @@ deb https://${1}@${REPO_URL}/ubuntu precise main
 EOF
 }
 
+is_package_installed() {
+    dpkg-query -s "$1" >/dev/null 2>&1
+}
+
 
 enable_esm() {
-    import_gpg_key
+    cp "${KEYRINGS_DIR}/${REPO_KEY_FILE}" "$APT_KEYS_DIR"
+    if ! is_package_installed apt-transport-https; then
+        apt-get install apt-transport-https
```

Comment:
I think we should use `apt-get install -y` here. As an improvement, we could wrap this install in a nice message saying "Installing missing dependency <foo>..." and only display apt's output in the case of an error, or else the screen gets too polluted in my opinion: see https://pastebin.canonical.com/187004/